jobs/probe_scraper.sh

```diff
@@ -7,17 +7,25 @@ BUCKET="net-mozaws-prod-us-west-2-data-pitmo"
 CACHE_DIR="probe_cache"
 OUTPUT_DIR="probe_data"
 
+ANACONDA_PATH=/mnt/anaconda2
```

Comment:
Maybe `anaconda3` since that's what's being installed?

Reply:
So this and the following comment (adding to the path) are both because I'm co-opting the existing files and path: https://github.com/mozilla/emr-bootstrap-spark/blob/master/ansible/files/bootstrap/telemetry.sh#L330.

That's probably not the best method, it should be fine to add this to `anaconda3` and point `PATH` to that first.